pandas_gbq/gbq.py

```diff
@@ -374,6 +379,26 @@ def process_http_error(ex):
 
         raise GenericGBQException("Reason: {0}".format(ex))
 
+    def download_table(
+        self, table_id, max_results=None, progress_bar_type=None, dtypes=None
+    ):
```

Comment:
(nit) Maybe completely annotate new methods?

Reply:
Good call, done.

I probably could have used `Dict[str, Any]` like we did [here](https://github.com/googleapis/python-bigquery/blob/66b3dd9f9aec3fda9610a3ceec8d8a477f2ab3b9/google/cloud/bigquery/job/query.py#L1561) for dtypes, but maybe something more specific would be better?

Note: we aren't currently running any sort of type checking in this repo. There's an open issue for it here: https://github.com/googleapis/python-bigquery-pandas/issues/325

Reply:
For a start, `Dict[str, any]` is probably just fine, especially if type checks are not implemented yet. Let's just make sure it's consistent with the docstring, the latter currently says `Optional[Map[str, Union[str, pandas.Series.dtype]]]`.